### PR TITLE
DOC: Rerun changelog generator

### PR DESCRIPTION
**Pull request by ericdill**
Amazingly good catch @jakirkham (this might sound sarcastic, but it truly is a sincere "amazingly")

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Change Log
 
 ## [0.7.2](https://github.com/maxpoint/conda-mirror/tree/0.7.2) (2017-05-23)
-
```
Comment (jakirkham) [inline CHANGELOG.md]: Do you still want this blank line? Not sure how you like to do formatting here. So this is merely a question.

**Comment by jakirkham** [comment]
np. Glad to help. 😄

File: CHANGELOG.md (hunk 1 above)
Comment (ericdill) [inline CHANGELOG.md]: The automated changelog generator removed it, so 🤷‍♂️ 

> Not sure how you like to do formatting here. 

I like to do none of the formatting and let the generator handle it 😀

Comment (jakirkham) [inline CHANGELOG.md]: Yeah, it seems to do it everywhere else and looks fine there. So seems fine. 😄

**Comment by jakirkham** [comment]
LGTM

**Comment by ericdill** [comment]
Thanks for the eyes @jakirkham

**Pull request merged**